modules-packages-crates/includes/7-knowledge-check.md

```diff
@@ -0,0 +1,33 @@
+# Knowledge check
+
+Why does the following code won't compile?
```

Comment:
@tilacog same comment here as in module 5. The question can't be more than a single line. Is there a way to simplify this or have a question without code?

Reply:
I'll keep this discussion open, just for me to keep track of it,  and resolve it the same way as in module 5, then.